### ISSUE TITLE
gitlab-runner@13.11.0: hash check failed

### ISSUE THREAD

**Issue by dyllan-to-yu** [ScoopInstaller/Main#2197]
```
URL:         https://gitlab-runner-downloads.s3.amazonaws.com/v13.11.0/binaries/gitlab-runner-windows-amd64.exe#/gitlab-runner.exe
First bytes: 4D 5A 90 00 03 00 04 00
Expected:    5f5ed5f9ccded4d04546720824470eae5efc4b2348712b9d5aae33fc7f5345d5
Actual:      c655401161fbe05577e4c5dd1068e2dfdf2cc363d71619b30ca2e72f982a70cf
```

I'm not sure where to find a source for the checksum for this :d

**Comment by dyllan-to-yu** [comment]
Dupe of  #2103  #2105 
Fixed by #2104 and #2198  ?

**Comment by akirayamamoto** [comment]
Fixed on https://github.com/ScoopInstaller/Main/commit/df20b73f4e0804caf80b1db30f3cea0bcbdd5dcd
